layout/paypal.ejs

```diff
@@ -0,0 +1,39 @@
+<form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="paypal">
```

Comment:
Ok, but I think in the future if we add ability to automagically renew, we should also make sure the flow works and add the sandbox.